src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -78,11 +78,13 @@ better or optimization algorithms require less adaptation.
 
 ## Model conditioning and curvature
 
-Because Stan's algorithms (other than Riemannian Hamiltonian Monte
-Carlo) rely on step-based gradient-based approximations of the density
+Because Stan's algorithms rely on step-based gradient-based
+approximations of the density
 (or penalized maximum likelihood) being fitted, posterior curvature
 not captured by this first-order approximation plays a central role in
 determining the statistical efficiency of Stan's algorithms.
+^[Riemannian Manifold Hamiltonian Monte Carlo (RMHMC) overcomes this difficulty by simulating the Hamiltonian dynamics in a space with a position-dependent metric; see @GirolamiCalderhead:2011 and @Betancourt:2012, but this must be done on the fly at every step, making this conditioning very computationally expensive.]
```

Comment:
Given this is the User's Guide, I suggest including a statement that Stan doesn't include RHMC!  I'd almost rather not include this, as it's no more relevant for Stan than other samplers.

Reply:
removed.